src/main/webapp/content/scss/themes/_dark-variables.scss

```diff
@@ -337,6 +337,13 @@ $quiz-scoring-info-student-modal-question-wrong: $warning;
 $quiz-question-background: $neutral-dark;
 $quiz-question-border-color: $neutral-dark-l-20;
 
+$quiz-visual-question-border-color: #d3d3d3;
+$quiz-visual-answer-correct-container-color: #53a351;
+$quiz-visual-answer-correct-container-hover-color: #90c48e;
+$quiz-visual-answer-wrong-container-hover-color: #f3837a;
+$quiz-visual-answer-add-container-color: #508ac7;
+$quiz-visual-answer-add-container-hover-color: lighten($quiz-visual-answer-add-container-color, 20%);
+
```

Comment:
One last thing, sorry.
Seems like you updated the definitions to default colors in the light theme only in 5ccb99481c94f136b5cf9529d4eace2429f46dfe. We need to do this for the dark theme, too.